local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
What that comment says :-)

Check on your local workspace - do you have a valid `.gitignore` and are you using Git normally? This file shouldn't be included